subsys/net/lib/aws_fota/src/aws_fota.c

```diff
@@ -417,6 +454,14 @@ int aws_fota_mqtt_evt_handler(struct mqtt_client *const client,
 		if (evt->result != 0) {
 			return 0;
 		}
+		if (evt->param.suback.message_id == SUBSCRIBE_NOTIFY_NEXT) {
+			printk("sub notify_next_topic\n");
```

Comment:
Should this be some log function instead?

Reply:
Yup that should have been a log function will fix it.